packages/material-ui/src/Select/Select.test.js

```diff
@@ -1112,4 +1112,23 @@ describe('<Select />', () => {
     );
     expect(document.activeElement).to.equal(getByRole('button'));
   });
+
+  it('should not override the event.target on click events', () => {
```

Comment:
It's a `MouseEvent` hence "mouse events". Just like it is a `TouchEvent` in the other test hence "touch events". Or change the description in Slider to "click events" and "touchstart events" respectively.